glide.yaml

```diff
@@ -0,0 +1,24 @@
+package: .
```

Comment:
`glide install` says

```
[WARN]	The name listed in the config file (.) does not match the current location (github.com/groovenauts/blocks-gcs-watcher)
```

so, `package: .` should be `package: github.com/groovenauts/blocks-gcs-watcher`